src/NServiceBus.Persistence.CosmosDB.TransactionalSession.AcceptanceTests/ConfigureEndpointCosmosDBPersistence.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.DependencyInjection;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting;
+using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.Configuration.AdvancedExtensibility;
+using NServiceBus.Persistence.CosmosDB;
+using NServiceBus.TransactionalSession.AcceptanceTests;
+
+public class ConfigureEndpointCosmosDBPersistence : IConfigureEndpointTestExecution
+{
+    public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
+    {
+        if (configuration.GetSettings().Get<bool>("Endpoint.SendOnly"))
```

Comment:
why is this needed?

Reply:
Aligned with the other Configure files in this repo

Reply:
Happy to remove though